common/apiClient.ts

```diff
@@ -0,0 +1,38 @@
+import { User } from './types'
+import Axios from 'axios'
+
+const host = 'https://application-portal.vercel.app'
```

Comment:
This works for prod, but won't work for dev (when the domain should be http://localhost:3000). We should dynamically figure out what the env is and export a domain constant based on that.

Something like the `domain` and `getEnv` and `isProd` functions [here](https://github.com/sandboxnu/office-hours/blob/master/packages/common/index.ts) is what we should do.

But I can do this in a separate PR!

Reply:
sounds good!